polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -858,13 +858,12 @@ populatePolygeistToLLVMTypeConversion(LLVMTypeConverter &typeConverter) {
           auto ST = LLVM::LLVMStructType::getIdentified(
               &typeConverter.getContext(), *type.getName());
           if (!ST.isInitialized()) {
-            if (failed(ST.setBody(convertedElemTypes, /*isPacked=*/false)))
+            if (failed(ST.setBody(convertedElemTypes, type.getIsPacked())))
```

Comment:
Nit: `getIsPacked()` is a bit non-intuitive, could we add `isPacked()`?

Reply:
done.